dataactcore/utils/report.py

```diff
@@ -9,18 +9,32 @@ def get_report_path(job, report_type):
     Return the filename for the error report.
     Does not include the folder to avoid conflicting with the S3 getSignedUrl method.
     """
-    path = 'submission_{}_{}_{}_report.csv'.format(job.submission_id, job.file_type.name, report_type)
-    return path
+    return report_file_name(job.submission_id, report_type == 'warning',
```

Comment:
If we're just forwarding the data to `report_file_name`, do still need this encompassing function as opposed to calling `report_file_name` directly?